esx_service/vmdk_ops.py

```diff
@@ -211,6 +220,14 @@ def validate_vsan_policy_name(policy_name, vmdk_path):
     if not vsan_policy.policy_exists(policy_name):
         raise ValidationError('Policy {0} does not exist'.format(policy_name))
 
+def validate_disk_allocation_format(format):
+    """
+    Ensure format is valid.
+    """
+    valid_opts = ["zeroedthick", "thin", "eagerzeroedthick"]
```

Comment:
I think it makes sense to keep choices ("valid_opts") in esx_service/volume_kv.py, since you do keep DEFAULT there already